### PR TITLE
feat: enable mainnet

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

Masternode owners have expressed interest in testing dashmate on mainnet.

## What was done?

- Add mainnet config with platform disabled
- Disable tenderdash init on mainnet

## How Has This Been Tested?

- Tested locally
- Ongoing testnet on VPS

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: templates/core/dash.conf.dot [hunk 1]
```diff
@@ -26,11 +26,12 @@ externalip={{=it.externalIp}}
 whitelist={{=it.externalIp}}
 {{?}}
 
-# Indices
+{{? it.network !== 'mainnet'}}# Indices
```
Comment (thephez) [inline templates/core/dash.conf.dot:29]: I thought indices were required on mainnet. Or is this because they're enforced automatically?

Comment (strophy) [reply templates/core/dash.conf.dot:29]: I think the necessary indices will be enforced automatically. I have tried syncing this branch on a 55GB Vultr VPS with following results:

- Indices enabled: drive full at block 1303209
- Indices disabled: sync complete using 37GB

Comment (thephez) [reply templates/core/dash.conf.dot:29]: I think txindex is the only one currently required by masternodes (attempting to confirm this). If that's accurate and platform requires additional indices, we'll probably need to leave them in here. 

Also, if that is the case, I suppose the next release of Core should add them to the list of enforced ones for masternodes. @PastaPastaPasta @UdjinM6

Comment (strophy) [reply templates/core/dash.conf.dot:29]: Yes, I agree. This branch is currently for some enthusiastic users to try setting up mainnet nodes with dashmate on their existing hardware (typically 55GB Vultr nodes). This condition check for mainnet can be removed once platform is supported on mainnet and these indices are required again.

Comment (kxcd) [reply templates/core/dash.conf.dot:29]: Yes, when you configure the blskey it automatically enables several other dependent options I believe the additional index is one of them and no doubt once Evo is launched to mainnet those additional indices will be added to the default list.

OT: It makes for an interesting question on upgrade from existing network to Evo, I suppose each exiting legacy mnode will have to reindex (expensive) their chains.

**Review by antouhou — DISMISSED** [review]

**Review by antouhou — APPROVED** [review]

**Pull request merged**